vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -161,15 +162,14 @@ def terminate_pool(self):
 
     def stop_pool(self):
         self.log.info("Stopping Thread Pool, waiting for queued jobs to finish")
-        for _ in self.pool._workers:
-            self.pool._workq.put(SENTINEL)
-        self.pool.close()
+        self.pool.terminate()
```

Comment:
This is not the same, `terminate` will also empty the queues, and we don't want that

Reply:
Thanks, fixed.